src/blocks/scratch3_procedures.js

```diff
@@ -13,46 +13,39 @@ class Scratch3ProcedureBlocks {
      */
     getPrimitives () {
         return {
-            procedures_definition: this.definition,
             procedures_call: this.call,
             argument_reporter_string_number: this.argumentReporterStringNumber,
             argument_reporter_boolean: this.argumentReporterBoolean
         };
     }
 
-    definition () {
-        // No-op: execute the blocks.
```

Comment:
@cwillisf Do you have any concerns for extensionification of custom procs that the procedure definition function will not exist?

In general we expect either `opcode` or both `opcode` and `func` to be defined for a given block right? We should make sure we start to handle these non-existent functions well.

Reply:
We already handle non-existent functions (hats like event_whenflagclicked and shadows). Do we instead need to handle confirming that non-existence is expected?